test/functionalTest/cases/1601_restrict_chars_in_ids/restrict_chars_attr.test

```diff
@@ -0,0 +1,258 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
As general comment for the .test files included in the PR, I think we should add also some test also for NGSIv1, to test the CLI switch.